### PR TITLE
[15.0][IMP] fieldservice_person_salary: new Planning section in fsm.order report

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
New Planning section in fsm.order report

Spanish example:
imagen

### CODE REVIEW

File: fieldservice_person_salary/__manifest__.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
         "views/fsm_order_views.xml",
         "views/fsm_order_salary_views.xml",
         "views/fsm_order_salary_menu.xml",
+        "reports/fsm_order_template.xml"
```
Comment (dalonsod) [inline fieldservice_person_salary/__manifest__.py]: Cosmetic change, missing comma:
```suggestion
        "reports/fsm_order_template.xml",
```

File: fieldservice_person_salary/reports/fsm_order_template.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                <h3>Planning</h3>
+                <div class="row">
+                    <div class="col-4">
+                        <span>Start Date: <span t-field="doc.scheduled_date_start"/></span>
+                    </div>
+                    <div class="col-4">
+                        <span>End Date: <span t-field="doc.scheduled_date_end"/></span>
+                    </div>
+                    <div class="col-4">
+                        <span>Duration: <span t-field="doc.scheduled_duration"/></span>
```
Comment (dalonsod) [inline fieldservice_person_salary/reports/fsm_order_template.xml]: Duration formatting. I think this widget doesn't require an extra addon dependency:
```suggestion
                        <span>Duration: <span t-field="doc.scheduled_duration" t-options="{'widget': 'duration', 'digital': True, 'unit': 'hour', 'round': 'minute'}"/></span>
```

File: fieldservice_person_salary/reports/fsm_order_template.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template
+        id="report_fsm_order_document_person_salary"
+        inherit_id="fieldservice.report_fsm_order_document"
+    >
+
+        <xpath expr="//div[@id='description']" position="after">
+            <div id="planning">
```
Comment (dalonsod) [inline fieldservice_person_salary/reports/fsm_order_template.xml:10]: An optional improvement should be hide the whole section in the report if all the fields of the sections are unset. Please comment this with project manager LR, I'm not sure what is the best option

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments. When attended, this could be merged IMO

File: fieldservice_person_salary/reports/fsm_order_template.xml (hunk 3 above)
Comment (ChristianSantamaria) [reply fieldservice_person_salary/reports/fsm_order_template.xml:10]: At the moment it is not necessary according to what he tells me

**Comment by ChristianSantamaria** [comment]
Let go to MERGEEE!!!

**Pull request merged**